src/plugins/is-bp-modal-enabled-mixin.js

```diff
@@ -0,0 +1,27 @@
+import { trackExperimentVersion, getExperimentSettingCached } from '#src/util/experiment/experimentUtils';
+
+export const HOME_BP_MODAL_EXP_KEY = 'home_page_bp_modal';
+
+export default {
+	data() {
+		return {
+			isBpModalEnabled: false
+		};
+	},
+	methods: {
+		initializeIsBpModalEnabledExp() {
```

Comment:
Two options.
1. Call this function on mounted in the page.
2. Change this logic to on mounted in the mixin.

Tested locally and BP side sheet doesn't open focing the experiment.